sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Infrastructure/EventProcessorManager.cs

```diff
@@ -273,7 +273,7 @@ public async Task WaitStabilization()
         ///   Filtering expired ownership is assumed to be responsibility of the caller.
         /// </remarks>
         ///
-        private bool AreOwnershipDistributionsTheSame(IEnumerable<PartitionOwnership> first,
+        internal static bool AreOwnershipDistributionsTheSame(IEnumerable<PartitionOwnership> first,
                                                       IEnumerable<PartitionOwnership> second)
```

Comment:
nit:  If you would be so kind, could you line up `second` with the declaration for `first`?